packages/design-system/src/components/VerticalNav/VerticalNav.test.tsx

```diff
@@ -137,48 +139,20 @@ describe('VerticalNav', () => {
         },
       ],
     };
-    const wrapper = mount(<VerticalNav {...props} />);
-    const parentWrapper = wrapper.findWhere((n) => n.prop('id') === 'parent').first();
-    const childWrapper = wrapper.findWhere((n) => n.prop('id') === 'child').first();
-    const grandchild1Wrapper = wrapper.findWhere((n) => n.prop('id') === 'grandchild-1').first();
-    const grandchild2Wrapper = wrapper.findWhere((n) => n.prop('id') === 'grandchild-2').first();
-    const fooWrapper = wrapper.findWhere((n) => n.prop('label') === 'Foo').first();
+    render(<VerticalNav {...props} />);
+    const parentEl = screen.getByText('Parent');
+    const childEl = screen.getByText('Child');
+    const grandchild1El = screen.getByText('Grandchild 1');
+    const grandchild2El = screen.getByText('Grandchild 2');
+    const fooEl = screen.getByText('Foo');
 
     // Parents and self are selected
-    expect(
-      parentWrapper
-        .find('.ds-c-vertical-nav__label')
-        .first()
-        .hasClass('ds-c-vertical-nav__label--current')
-    ).toBe(true);
-
-    expect(
-      childWrapper
-        .find('.ds-c-vertical-nav__label')
-        .first()
-        .hasClass('ds-c-vertical-nav__label--current')
-    ).toBe(true);
-
-    expect(
-      grandchild1Wrapper
-        .find('.ds-c-vertical-nav__label')
-        .first()
-        .hasClass('ds-c-vertical-nav__label--current')
-    ).toBe(true);
+    expect(parentEl.classList).toContain('ds-c-vertical-nav__label--current');
```

Comment:
Would this be a good test to check for the `aria-current` property? I believe that's a recent addition and might not be covered by original unit tests.